pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -97,6 +97,11 @@ func (s *AsyncAgentStore) Store(key AgentCommand, value interface{}) {
 			domainInfo.FSFreezeStatus = &status
 		}
 
+		// The event should always have the updated interfaces, otherwise, the vmi status report  may be broken
```

Comment:
Beyond just the interfaces, I think we should send the full domainInfo object, with OSInfo, Interfaces, and FSFreezeStatus on every update. The receiving side of this doesn't even look at the `key` to see what changed. This will continue to cause subtle issues for us in the future as more agent information is polled for if we're sending partial info.

Reply:
I agree. The PR was updated to always send the full domain info,